moped-editor/package.json

```diff
@@ -11,7 +11,7 @@
     "start:staging": "env-cmd -e staging npm run start:local",
     "start:production": "env-cmd -e production npm run start:local",
     "build": "react-scripts build",
-    "build:local": "GENERATE_SOURCEMAP=false react-scripts build && cp _redirects build && cp -r build moped && mv moped build",
+    "build:local": "grep 'kickstand' package-lock.json && GENERATE_SOURCEMAP=false react-scripts build && cp _redirects build && cp -r build moped && mv moped build",
```

Comment:
I think you mentioned why you added this `grep` in dev sync but can you explain it again for me? I thought using the commit hash instead of branch name turned out to be the solution?

Reply:
Ah, yes this was to display the exact version of kickstand the build downloaded. It seems that Netlify ignores the package-lock.json file and overwrites it with whatever npm finds in `npm install`.

So I found out that Netlify kept downloading the wrong version using `kickstand@main` and switched to `kickstand@<commit id>`